torch_glow/src/FusingOptimizer.cpp

```diff
@@ -0,0 +1,49 @@
+#include "FusingOptimizer.h"
```

Comment:
* Add Copyright
* note that this was orignially from `pytorch/tvm` (right?)
* Is this something that should/could be added to PyTorch directly instead?

Reply:
Yeah this is copied from tvm. This is actually a temporary solution, since soon or later they will have linear support inside pytorch.